app/views/groups/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="restaurant-card-container">
+    
+        <div class="card mb-3">
+            <div class="restaurant-card-main row">
+            <div class="square-container col-auto">
+                <div class="square-image img-fluid">
+                    <img src="<%= @group.icon %>" class="img-fluid" alt="restaurantlogo">
+                </div>
+            </div>
+            <div class="restaurant-card-right col">
+                <div class="card-body">
+                    <div class="restaurant-title-container">
+                        <h5 class="card-title"><%= @group.name %></h5>
+                        
+                        <p>$<%=@sum_of_amount%></p>
+                        
+                    </div>
+                <p class="card-text"><small class="text-body-secondary"><%= @updated_time.strftime('%e %B %Y') %></small></p>
+                </div>
+            </div>
+            </div>
+        </div>
+        <%= link_to 'Delete', group_path, class: "btn btn-danger mb-2",method: :delete, data: {confirm: "Are you sure you want to delete Restaurant and its food items"} %>
+        <%= link_to 'Edit', edit_group_path, class: "btn btn-info mb-2" %>
+    
+        <% @group.foods.each do |food|%>
```

Comment:
[OPTIONAL]
- [ ] Good job displaying the group foods :clap: However it would be nice to use the instead variable **`@foods`** that you created in the groups controller
```suggestion
        <% @foods.each do |food|%>
```

Reply:
Thanks @LionRouge1, the changes asked by you has been done. 😃